packages/cascara/src/ui/Table/fixtures/TableStyleTest.module.scss

```diff
@@ -1,10 +1,11 @@
 :global {
```

Comment:
This is not how the styles are actually going to be attached to all of these table elements. This was initially just for that test. We can still leave this way for now... but we actually want to update these to apply them directly as CSS Modules to each of the table elements.

These items should probably also get renamed with classes to set those styles accordingly. That can be a separate task though: https://espressive.atlassian.net/browse/DEV-12472